STABLE_RELEASES.md

```diff
@@ -0,0 +1,147 @@
+# Stable Releases
+
+The following release series are currently supported and receieve bugfixes:
+
+* **0.37**
+* **0.39 "Launchpad"**
+
+Stable releases continue to receieve bugfixes until they reach End Of Life.
+The `0.37.x` release series will continue receiving bugfixes until the Cosmos Hub
+migrates to a newer release of the Cosmos-SDK.
+
+The `0.38.x` release series will reach End of Life once the `0.39.0` release is published.
```

Comment:
Don't we support the previous major version? Until we know no one is using 0.38 I think we should backport only bug fixes and security related things. In this scenario 0.38 would reach the end of life would be when 0.40 comes out

Reply:
`0.39.0` will be labeled `Launchpad` and that is the release we intend to support until 6 months after 0.40 is out. I don't think we have enough capacity to support 0.37, 0.38 and 0.39 - but I may well be wrong here. Thoughts @alexanderbez @clevinson?

Reply:
once 0.40 is out we drop support for 0.38 and really should for 0.37 as well but that is blocked on the hub..

I thought this is how we have been doing things, we only support the latest major version behind the latest. so when 0.39 comes we only support 0.38 then 0.40 comes and 0.38 reaches end of life. unless we explicitly say that 0.38 is unsafe then it would reach end of life sooner.

Reply:
Plan changed. the 0.38 series will be discontinued once _0.39.0_ is out, clients will be asked to upgrade their applications if they intend to receieve backports and updates. As a matter of fact, the _0.39_ release series is going to carry the Launchpad codename and should be maintained long enough to give people reasonable time to migrate to 0.40

Reply:
That sounds good to me. Might need to update the wording since this line is the first time 0.38 is referenced?

Reply:
I've updated the doc, please review again